src/sql/workbench/contrib/dashboard/browser/pages/databaseDashboardPage.contribution.ts

```diff
@@ -96,12 +96,22 @@ export const databaseDashboardSettingSchema: IJSONSchema = {
 			},
 			widget: {
 				'tasks-widget': [
-					'newQuery',
-					'mssqlCluster.task.newNotebook',
 					{ name: 'backup', when: '!mssql:iscloud && mssql:engineedition != 11' },
-					{ name: 'restore', when: '!mssql:iscloud && mssql:engineedition != 11' }
+					{ name: 'restore', when: '!mssql:iscloud && mssql:engineedition != 11' },
+					'newQuery',
+					'mssqlCluster.task.newNotebook'
```

Comment:
the order should be new query -> new notebook -> backup -> restore

Reply:
I originally suggested suggested Backup > Restore > New Query > New Notebook because I thought backup/restore might be more common actions on a dashboard since there are several other entry points to create a new query or new notebook — but it's really hard to claim this without data to back it up.

Let's follow Alan's suggestion above for now since it matches the ordering of our Connections right-click menu. Then let's get some telemetry on this when we can.